spring-restdocs/src/main/java/org/springframework/restdocs/hypermedia/HypermediaDocumentation.java

```diff
@@ -59,5 +63,47 @@ public static LinkSnippetResultHandler documentLinks(String outputDir,
 		return new LinkSnippetResultHandler(outputDir, linkExtractor,
 				Arrays.asList(descriptors));
 	}
+	
+	/**
+	 * Creates a {@code FieldDescriptor} that describes a field with the given {@code name}.
+	 * 
+	 * @param name The name of the field
+	 * @return a {@code FieldDescriptor} ready for further configuration
+	 * @see RestDocumentationResultHandler#withRequestFields(FieldDescriptor...)
+	 * @see RestDocumentationResultHandler#withResponseFields(FieldDescriptor...)
+	 */
+	public static FieldDescriptor fieldWithName(String name) {
```

Comment:
I wonder if this should be a path rather just a name? It would enable the documentation of more complex nested structures.